scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxy.java

```diff
@@ -99,12 +99,12 @@ public RMState getState() {
         return proxyActiveObject.getState();
     }
 
-    public void rebind(URI rmURI) throws RMException, RMProxyCreationException {
+    public boolean rebind(URI rmURI) throws RMException, RMProxyCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)